src/components/view/Navigation/index.js

```diff
@@ -36,20 +32,10 @@ class Navigation extends React.Component {
         super(props);
         this.navigateToRoute = this.navigateToRoute.bind(this);
         this.themePalette = this.props.muiTheme.palette;
+        this.state = {open: false};
     }
-    
-    buildIconMenu (actions) {
-        const { logout } = actions;
-        return (
-            <IconMenu
-                iconButtonElement={<IconButton><ExpandMoreIcon /></IconButton>}
-                anchorOrigin={{ horizontal: 'left', vertical: 'top' }}
-                targetOrigin={{ horizontal: 'left', vertical: 'top' }}
-            >
-                <MenuItem onTouchTap={logout} primaryText="Logout" />
-            </IconMenu>
-        );
-    };
+
+    handleToggle = () => this.setState({open: !this.state.open});
```

Comment:
Use function definition syntax as `navigateToRoute` instead of prop definition syntax. Use line breaks between statements when the function definition body has more than one statement.